### PR TITLE
Add download and usage instructions

### PR DESCRIPTION
**Pull request by ejona86**

### CODE REVIEW

**Comment by ejona86** [comment]
@zhangkun83

File: README.md [hunk 1]
```diff
@@ -7,6 +7,56 @@ gRPC-Java works with JDK 6. TLS usage typically requires using Java 8, or Play
...
+```
+compile 'io.grpc:grpc-all:0.7.0'
+```
+
+[the JAR]: https://search.maven.org/remote_content?g=io.grpc&a=grpc-all&v=0.7.0
+
+Development snapshots are available in [Sonatypes's snapshot
+repository](https://oss.sonatype.org/content/repositories/snapshots/).
+
+For protobuf-based codegen, use [protobuf-gradle-plugin][]:
```
Comment (zhangkun83) [inline README.md]: We probably want to add a link to http://sergei-ivanov.github.io/maven-protoc-plugin/examples/protoc-plugin.html for Maven users.

Comment (ejona86) [inline README.md]: Very good point. Done.

**Comment by zhangkun83** [comment]
LGTM

**Pull request merged**